less/admin/AdminNav.less

```diff
@@ -20,60 +94,84 @@
     width: @admin-pane-width;
     .box-shadow(0 6px 6px @shadow-color);
     background: @body-bg;
-    border-top: 1px solid @control-bg;
     z-index: @zindex-pane;
-    overflow: auto;
+    overflow-y: scroll;
+    padding-bottom: 40px;
 
     .affix & {
       position: fixed;
       bottom: 0;
-      height: auto;
     }
   }
   .App-content .sideNavOffset {
     margin-left: @admin-pane-width;
   }
   .App-nav .AdminNav {
-    .Dropdown-menu > li {
-      > a {
-        padding: 15px 15px 15px 45px;
-        display: block;
-        text-decoration: none;
-        white-space: normal;
+    .Dropdown-menu {
+      .item-search {
+        margin-top: 10px;
+        margin-bottom: 20px;
       }
-      > a, > a:hover, &.active > a {
-        color: @muted-color;
-      }
-      > a:hover {
-        background: @control-bg;
-      }
-      &.active > a {
-        background: @control-bg;
-        font-weight: normal;
 
-        .Button-label, .Button-icon {
+      > li {
+        > a {
+          padding: 10px 10px 10px 45px;
+          display: block;
+          text-decoration: none;
+        }
+        > a,
+        > a:hover,
+        &.active > a {
           color: @text-color;
-          font-weight: bold;
+        }
+        > a:hover {
+          background: @control-bg;
+        }
+        &.active > a {
+          background: @primary-color;
+          font-weight: normal;
+
+          .Button-label,
+          .Button-icon {
+            filter: invert(1);
```

Comment:
I think it'd be better to use `color: @body-bg` on the anchor above it instead of the filter. More consistent with the rest of the codebase and ensures we're getting the correct color.